tests/config_test.go

```diff
@@ -46,6 +46,7 @@ import (
 	"kubevirt.io/kubevirt/tests/decorators"
 	"kubevirt.io/kubevirt/tests/exec"
 	"kubevirt.io/kubevirt/tests/framework/kubevirt"
+	"kubevirt.io/kubevirt/tests/libconfigmap"
```

Comment:
Commit message: Do you mean `libconfigmap`?

Reply:
Nice catch, thanks